sandbox/install-client-archives.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+set -e
+
+# To run this script, you must first set the version number.
+# For more information, see README.md
+
+if [[ -z "${VERSION}" ]]; then
+    echo "You must set the release version number. For example:"
+    echo "export VERSION=0.2.31" 1>&2
+    exit 1
+fi
+
+CLIENT_PACKAGES=( cueadmin cuegui cuesubmit pycue pyoutline )
+
+BASE_URL=https://github.com/AcademySoftwareFoundation/OpenCue/releases/download/v
+
+mkdir opencue-downloads
+
+cd opencue-downloads
+
+for PACKAGE in "${CLIENT_PACKAGES[@]}"; do
+    wget ${BASE_URL}${VERSION}/${PACKAGE}-${VERSION}-all.tar.gz
```

Comment:
Is `wget` included with macOS by default? I seem to remember that it's not, and while I don't have a fresh mac on hand to test this out with, my workstation seems to only have a homebrew-installed one:

```
$ which -a wget
/Users/cipriano/homebrew/bin/wget
```

Should we add it to the mac quickstart?

Reply:
I don't believe it's included by default. I don't have it on any of my machines.

Yes, agree about adding it to the mac quick start. I plan to update the quick starts over the next couple of days so that you can review the two PRs side-by-side. I wanted to get your initial reaction to this approach before updating the docs. Thanks for the quick review and the enthusiastic response.

Reply:
Done.